pallets/automation-time/src/lib.rs

```diff
@@ -593,12 +593,10 @@ pub mod pallet {
 			encoded_call: Vec<u8>,
 			encoded_call_weight: Weight,
 		) -> DispatchResult {
-			// Remove below directive when implemented
-			#![allow(unused_variables)]
-			let _who = ensure_signed(origin)?;
-
-			Err(Error::<T>::TaskNotSupported)?;
+			let who = ensure_signed(origin)?;
+			let action = Action::XCMP { para_id, currency_id, encoded_call, encoded_call_weight };
 
```

Comment:
could verify `currency_id === Native` here but an error will also throw when `get_xcm_fee` is called.

Reply:
ya let all that logic happen downstream as we need to allow other tokens soon